compiler/ballerina-lang/src/main/java/io/ballerina/projects/CompilationOptionsBuilder.java

```diff
@@ -24,7 +24,7 @@
  *
  * @since 2.0.0
  */
-class CompilationOptionsBuilder {
+public class CompilationOptionsBuilder {
```

Comment:
Shall we make the setters package-private other that for `offline`?

Reply:
Done